libcontainer/cgroups/systemd/v1.go

```diff
@@ -28,6 +28,16 @@ func NewLegacyManager(cg *configs.Cgroup, paths map[string]string) (cgroups.Mana
 	if cg.Rootless {
 		return nil, errors.New("can't use rootless systemd cgroups manager on cgroup v1")
 	}
+	if cg.Resources != nil && cg.Resources.Unified != nil {
+		return nil, cgroups.ErrV1NoUnified
+	}
+	if paths == nil {
```

Comment:
Looking for potential corner-cases: is an empty `map[string]string` to be considered different for this code path, or should it be treated the same? (if "the same", we'd have to check for `len(paths)`)

Reply:
Forget to reply -- I thought about that and I guess the non-nil map (even when empty) should be handled as it is right now, IOW it means "paths are set, no need to guess any". Not sure if/when this can be useful but theoretically it kind of makes sense.